src/spaces/openstreetmap.jl

```diff
@@ -951,6 +951,26 @@ function Agents.nearby_ids(
     return nearby
 end
 
+function Agents.nearby_ids(
```

Comment:
I haven't understood why this function exists. THere is already a version for this function, right?

Reply:
We talked about this in another comment thread (https://github.com/JuliaDynamics/Agents.jl/pull/934#discussion_r1403802913). Maybe you'd like to chime in over there how we would like to handle this case?